app/policies/story_policy.rb

```diff
@@ -7,6 +7,10 @@ def show?
     is_admin? || is_project_member? && current_project.stories.find_by_id(record.id)
   end
 
+  def story?
```

Comment:
Instead of creating a new method to do the same thing than `show?`, is not better to `aliasing` it?